colossalai/inference/batch_bucket.py

```diff
@@ -447,3 +447,6 @@ def get_sequence_lengths(self) -> torch.Tensor:
     def fd_inter_tensor(self) -> None:
         assert self.fd_interm_tensor is not None, "fd_interm_tensor is not provided"
         return self.fd_interm_tensor
+
+    def __repr__(self) -> str:
+        return f"(sequences_dict={self._sequences_dict}, is_prompts={self.is_prompts})"
```

Comment:
Is it supposed to be used only in testing?

Reply:
I have added logger.debug in test_batch_bucket.py